WordPress/Classes/ViewRelated/System/FancyAlertViewController+QuickStart.swift

```diff
@@ -33,14 +33,14 @@ extension FancyAlertViewController {
 
         let notNowButton = ButtonConfig(Strings.notNowText) { controller, _ in
             controller.dismiss(animated: true)
-
+            QuickStartTourGuide.deletePendingLocalNotifications()
```

Comment:
In case you create a new site but you have already a pending notification. This might happen if you completed a task and then you create a new site skipping the QS.